.prettierrc.json

```diff
@@ -0,0 +1,2 @@
+// This file holds options
```

Comment:
Does this have a .js or extensionless option? I’m not sure json supports comments